src/Network/Network/Common/Utils.cs

```diff
@@ -83,4 +85,180 @@ public static bool IsRegionRestrictedForBasicFirewall(string region)
             return false;
         }
     }
+
+    namespace Utilities.CIDR.IPv4
```

Comment:
Could you explain why writing a nested namespace here rather than creating a new file? And the naming may also cause confusion.

From the [Namespace Naming Guidelines](http://msdn.microsoft.com/en-us/library/893ke618(vs.71).aspx):

The general rule for naming namespaces is to use the company name followed by the technology name and optionally the feature and design as follows.

CompanyName.TechnologyName[.Feature][.Design]

Reply:
Good call. The utility functions don't really need more namespaces, so I've migrated them back into a static class similar to the existing ones in that file.